CONTRIBUTING.md

```diff
@@ -27,6 +27,7 @@ The rules build on several exisiting rulesets such as the airbnb ruleset, the es
 
 When the command is run it will print a JSON representation of each of the maintained rulesets. In addition the command will print the result of some audits that try to detect patterns we want to avoid, such as rules configured with a `warn` severity level.
 
+<!--
```

Comment:
I think we should migrate to using [beachball](https://microsoft.github.io/beachball/) for handling the release workflow as we have done in the design system. Has added benefits of enabling good workflows for CHANGLOGS and for avoiding manual tag publish steps.

I'll create an issue to implement the workflow and to update these commented out docs. For now the release workflow will be pretty manual requiring manually bumping versions across all the packages and manually publishing to npm. I imagine that will only be necessary for the first publish (as manual steps are needed to publish a new package anyway) and follow-up publishes will be with beachball integrated.